inference-engine/ie_bridges/c/tests/ie_c_api_test.cpp

```diff
@@ -281,6 +282,46 @@ TEST(ie_core_read_network, networkRead) {
     ie_core_free(&core);
 }
 
+TEST(ie_core_read_network_from_memory, networkReadFromMemory) {
+    ie_core_t *core = nullptr;
+    IE_ASSERT_OK(ie_core_create("", &core));
+    ASSERT_NE(nullptr, core);
+
+    std::string xml_content;
+    {
+        std::ifstream xml_file(xml);
+        while (xml_file)
+        {
+            char ch;
+            xml_file >> ch;
+            xml_content += ch;
+        }
```

Comment:
can we read the content in more optimal way? Like move to the end of file, understand the file size using ifstream::tellg and read all the content at once?

Reply:
Sure i will change that romorrow